pkg/apis/vpc/v1alpha1/subnet_types.go

```diff
@@ -119,14 +120,25 @@ type SubnetAdvancedConfig struct {
 	EnableVLANExtension bool `json:"enableVLANExtension,omitempty"`
 }
 
+// Additional DHCP server config for a VPC Subnet.
+// The additional configuration must be cleared when the Subnet has DHCP relay enabled or DHCP is deactivated.
+type DHCPServerAdditionalConfig struct {
+	// Reserved IP ranges.
```

Comment:
Do we need to mention the supported format like `Supported formats include: ["192.168.1.1", "192.168.1.3-192.168.1.100"]`

Reply:
thanks, added this.